src/classes/cooperative_AreaFarmersChart.cls

```diff
@@ -0,0 +1,86 @@
+public with sharing class cooperative_AreaFarmersChart {
+
+    private ApexPages.StandardController stdController;
+    private LAC_Cooperative__c cooperative;
+    public Decimal maxFarmArea;
+    
+    public Cooperative_AreaFarmersChart(ApexPages.StandardController controller) {
+        if (controller <> null){
+            this.stdController = controller;
+            this.cooperative = (LAC_Cooperative__c)controller.getRecord();
+            getMaxFarmArea();       
+        }                                         
+    }
+    
+    /**
+    *Get the maximum farm area related to a cooperative
+    *
+    */
+    public void getMaxFarmArea(){
+        maxFarmArea = (Decimal) [Select MAX(Farm_Area__c) fArea 
+                                From LAC_Farmer_Details__c 
+                                Where cooperativeProfile__c =:cooperative.Id][0].get('fArea');
+        if (maxFarmArea == null) maxFarmArea = 0;
+    }
+
+    /**
+    *Get an area list related to a farmers that belongs a specific cooperative
+    *
+    */
+    public List<LAC_Farmer_Details__c> getAreaFarmers() {
+        List<LAC_Farmer_Details__c> AreaFarmers= [Select Farmer_Code__c, 
```

Comment:
variables should start with small laters: areaFarmers not AreaFarmers. Classes would start with capital letters.